Dockerfile.test.cpu

```diff
@@ -50,32 +50,35 @@ RUN if [[ ${MPI_KIND} == "OpenMPI" ]]; then \
         wget -O /tmp/openmpi-3.0.0-bin.tar.gz https://github.com/horovod/horovod/files/1596799/openmpi-3.0.0-bin.tar.gz && \
             cd /usr/local && tar -zxf /tmp/openmpi-3.0.0-bin.tar.gz && ldconfig && \
             echo "mpirun -allow-run-as-root -np 2 -H localhost:2 -bind-to none -map-by slot -mca mpi_abort_print_stack 1" > /mpirun_command; \
-    elif [[ ${MPI_KIND} == "MLSL" ]]; then \
-        wget -O /tmp/l_mlsl_2018.3.008.tgz https://github.com/intel/MLSL/releases/download/IntelMLSL-v2018.3-Preview/l_mlsl_2018.3.008.tgz && \
-            cd /tmp && tar -zxf /tmp/l_mlsl_2018.3.008.tgz && \
-            /tmp/l_mlsl_2018.3.008/install.sh -s -d /usr/local/mlsl && \
-            wget https://raw.githubusercontent.com/intel/MLSL/master/mpirt_2019/include/mpi.h -P /usr/local/mlsl/intel64/include && \
-            wget https://raw.githubusercontent.com/intel/MLSL/master/mpirt_2019/include/mpio.h -P /usr/local/mlsl/intel64/include && \
-            wget https://raw.githubusercontent.com/intel/MLSL/master/mpirt_2019/include/mpicxx.h -P /usr/local/mlsl/intel64/include && \
-            wget https://raw.githubusercontent.com/AlekseyMarchuk/MLSL/master/mpirt_2019/bin/mpicc -P /usr/local/mlsl/intel64/bin && \
-            chmod +x /usr/local/mlsl/intel64/bin/mpicc && \
-            wget https://raw.githubusercontent.com/AlekseyMarchuk/MLSL/master/mpirt_2019/bin/mpicxx -P /usr/local/mlsl/intel64/bin && \
-            chmod +x /usr/local/mlsl/intel64/bin/mpicxx && \
-            wget https://raw.githubusercontent.com/AlekseyMarchuk/MLSL/master/mpirt_2019/bin/mpigcc -P /usr/local/mlsl/intel64/bin && \
-            chmod +x /usr/local/mlsl/intel64/bin/mpigcc && \
-            wget https://raw.githubusercontent.com/AlekseyMarchuk/MLSL/master/mpirt_2019/bin/mpigxx -P /usr/local/mlsl/intel64/bin && \
-            chmod +x /usr/local/mlsl/intel64/bin/mpigxx && \
-            wget https://raw.githubusercontent.com/AlekseyMarchuk/MLSL/master/mpirt_2019/lib/libmpicxx.so -P /usr/local/mlsl/intel64/lib && \
-            chmod +x /usr/local/mlsl/intel64/lib/libmpicxx.so && \
-            echo ". /usr/local/mlsl/intel64/bin/mlslvars.sh \"thread\"; \
+    elif [[ ${MPI_KIND} == "CCL" ]]; then \
+        wget -O /tmp/oneccl.tar.gz https://github.com/intel/oneccl/archive/master.tar.gz && \
+            cd /tmp && tar -zxf oneccl.tar.gz && \
+            mkdir oneccl-master/build && cd oneccl-master/build && cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local/oneccl -DCMAKE_BUILD_TYPE=Release && make -j install && \
+            wget -O /tmp/extra_lib.tar.gz https://github.com/ksenyako/oneccl/archive/related_lib.tar.gz && \
```

Comment:
For testing purpose those files were located in personal repo because it was not clear till this moment which files exactly should be added to the oneCCL repo. And what changes should be implied. It will be changed in the next commit